ztp/source-crs/SriovNetwork.yaml

```diff
@@ -6,14 +6,14 @@ metadata:
   annotations:
     ran.openshift.io/ztp-deploy-wave: "100"
 spec:
-  resourceName: ""
+#  resourceName: ""
```

Comment:
Why do these need to be commented out?

Reply:
cause it will be added as empty value field , no need to do that.